velox/functions/lib/DateTimeFormatterBuilder.cpp

```diff
@@ -56,6 +56,13 @@ DateTimeFormatterBuilder& DateTimeFormatterBuilder::appendWeekOfWeekYear(
   return *this;
 }
 
+DateTimeFormatterBuilder& DateTimeFormatterBuilder::appendWeekOfMonth(
```

Comment:
Should this function be called in buildJodaDateTimeFormatter or buildMysqlDateTimeFormatter? Like https://github.com/facebookincubator/velox/blob/main/velox/functions/lib/DateTimeFormatter.cpp#L1463.

Reply:
No. "Week Of Month" is not supported in [Joda](https://www.joda.org/joda-time/key_format.html) and [Mysql](https://www.w3schools.com/sql/func_mysql_date_format.asp).